js/components/DashTabs.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import { CSSTransitionGroup } from "react-transition-group";
+
+import ScriptChart from "./ScriptChart";
+import MiradorContainer from "./MiradorContainer";
+
+import "./DashTabs.css";
+
+const Tab = props => {
+  const { name } = props.tab;
+  const { activeTab, changeActiveTab } = props;
+
+  return (
+    <li
+      className={name === activeTab ? "is-active" : ""}
+      onClick={() => changeActiveTab(name)}
+    >
+      <a>
+        <span className="icon is-small">
+          <i className={props.icon} />
+        </span>
+        <span>{name}</span>
+      </a>
+    </li>
+  );
+};
+
+class Tabs extends React.Component {
+  static propTypes = {
+    tabList: PropTypes.array.isRequired,
+    activeTab: PropTypes.string,
+    changeActiveTab: PropTypes.func
+  };
+
+  render() {
+    return (
+      <div className="tabs is-toggle is-boxed">
+        <ul>
+          {this.props.tabList.map(tab => (
+            <Tab
+              tab={tab}
+              key={tab.name}
+              icon={tab.icon}
+              activeTab={this.props.activeTab}
+              changeActiveTab={this.props.changeActiveTab}
+            />
+          ))}
+        </ul>
+      </div>
+    );
+  }
+}
+
+const ActiveTabContent = props => <div>{props.content}</div>;
+
+const tabList = [
+  {
+    name: "Scriptchart",
+    icon: "fa fa-table",
+    content: <ScriptChart />
+  },
+  {
+    name: "Manuscript Viewer",
+    icon: "fa fa-image",
+    content: <MiradorContainer />
+  }
+];
+
+class DashTabs extends React.Component {
```

Comment:
We should probably break this into multiple files since it is technically multiple components. If we can have one component per file, we can more easily manage tests and knowing where everything is.

Reply:
Ah right -- so probably one component for DashTabs and one for just 'Tabs'. This is how it will be in the new PR unless you have other preferences.